### PR TITLE
Add min/max values for ints

### PR DESCRIPTION
**Pull request by kruai**
Add some min/max values for ints to address [RHCLOUD-14535](https://issues.redhat.com/browse/RHCLOUD-14535).
These values are probably overkill, but it's to address a bug found in testing. If you'd like to limit the values for specific fields, please address those in a separate PR.

HBI PR here: https://github.com/RedHatInsights/insights-host-inventory/pull/922

### CODE REVIEW

File: tests/utils/invalids.py [hunk 1]
```diff
@@ -55,9 +55,11 @@
     {"rhc_config_state": "x"*12},
     {"cpu_model": "x"*101},
     {"number_of_cpus": "35465"},
+    {"number_of_cpus": 4294967296},
```
Comment (ryandillinfelton) [inline tests/utils/invalids.py:58]: Could you also add invalids for the other fields you introduced limits on?
```suggestion
    {"number_of_cpus": 4294967296},
    {"number_of_sockets": 4294967296},
    {"cores_per_socket": 4294967296},
```

File: tests/utils/invalids.py [hunk 2]
```diff
@@ -55,9 +55,11 @@
...
     {"number_of_sockets": "35465"},
     {"cores_per_socket": "35465"},
     {"system_memory_bytes": "35465"},
+    {"system_memory_bytes": 18446744073709551617},
```
Comment (ryandillinfelton) [inline tests/utils/invalids.py:62]: Same here but just for MTU
```suggestion
    {"system_memory_bytes": 18446744073709551617},
    {"mtu": 18446744073709551617},
```

**Review by ryandillinfelton — CHANGES_REQUESTED** [review]
Just want some more robust testing. I think you can just commit the suggestions if you have no objection to them.

**Review by thearifismail — APPROVED** [review]
LGTM

**Comment by ryandillinfelton** [comment]
Looks like the `mtu` test failed :thinking:

File: tests/utils/invalids.py [hunk 3]
```diff
@@ -55,9 +55,14 @@
...
     {"cpu_model": "x"*101},
     {"number_of_cpus": "35465"},
+    {"number_of_cpus": 4294967296},
+    {"number_of_sockets": 4294967296},
+    {"cores_per_socket": 4294967296},
     {"number_of_sockets": "35465"},
     {"cores_per_socket": "35465"},
     {"system_memory_bytes": "35465"},
+    {"system_memory_bytes": 18446744073709551617},
+    {"mtu": 18446744073709551617},
```
Comment (ryandillinfelton) [inline tests/utils/invalids.py:65]:
```suggestion
    {"network_interfaces": [{"mtu": 18446744073709551617}]},
```

**Review by ryandillinfelton — CHANGES_REQUESTED** [review]
I think this'll fix the test issue

File: tests/utils/invalids.py (hunk 3 above)
Comment (ryandillinfelton) [reply tests/utils/invalids.py:65]: My bad with the faulty suggestion. Didn't notice mtu is a subfield

**Review by ryandillinfelton — APPROVED** [review]
Cool! Tests passing now!

**Pull request merged**